pkgs/applications/misc/fontpreview/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper, xdotool, fzf, imagemagick, sxiv }:
+
+stdenv.mkDerivation rec {
+  pname = "fontpreview";
+  version = "1.0.6";
```

Comment:
1.0.7 exists as commit, but no tag :(

https://github.com/sdushantha/fontpreview/commit/e18f6d39b35aa5658dba324cab9d8209b5036664

Reply:
Yep, I also noticed that. To me that means the author didn't intend to release the new version yet, but only wanted to prepare the new release.

Reply:
that's possible